src/Generator/Generators/CSharp/CSharpTextTemplate.cs

```diff
@@ -922,9 +922,10 @@ private void GenerateIndexerSetter(Function function)
         }
 
         private void GeneratePropertyGetter<T>(QualifiedType returnType, T decl,
-            Class @class, bool isAbstract = false)
+            Class @class, bool isAbstract = false, Class OrigClass = null)
```

Comment:
Please use lower case for "OrigClass". You can actually rename it to "originalClass" for more clarity.